Enable alert forwarding to Fluentd for Analysisd (#6691)
wazuh/wazuh-documentation: source/user-manual/manager/fluent-forwarder.rst

    2019-03-28 14:47:40.000000000 +0200 debug.test: "message"

Example using analysisd
-----------------------

This example is for testing purposes on a Debian machine, with the Wazuh manager installed.

Given the following configuration:

.. code-block:: xml

    <fluent-forward>
      <enabled>yes</enabled>
      <tag>debug.test</tag>
      <socket_path>/var/ossec/var/run/fluent.sock</socket_path>
      <address>localhost</address>
      <port>24224</port>
    </fluent-forward>

.. note::
    The ``socket_path`` must be relative or under the ``/var/ossec`` directory.

Set up the ``socket`` for analysisd:

.. code-block:: xml

    <socket>
      <name>fluent_socket</name>
      <location>/var/ossec/var/run/fluent.sock</location>
      <mode>udp</mode>
    </socket>

.. note::
    The ``location`` must have the same value as the ``socket_path`` set before.

> Set up a ``target`` to read from:

Reviewer: ```suggestion
Set up a target to read from:
```

.. code-block:: xml

    <global>
      <forward_to>fluent_socket</forward_to>
    </global>

.. note::
    In ``forward_to`` specify the ``name`` of the generated socket.

On a terminal, run the following commands as root to start a Fluentd server:

.. code-block:: console

    apt-get install -y ruby ruby-dev
    gem install fluentd
    fluentd -s conf
    fluentd -c conf/fluent.conf

Restart the Wazuh manager:

.. code-block:: console

    systemctl restart wazuh-manager

> When a new event is trigger to analysisd, the event is sent to fluentd is JSON format. You should see the message on the Fluentd server:

Reviewer: ```suggestion
When an event triggers an alert, it's sent to fluentd in JSON format. On the Fluentd server, you can see a message similar to this:
```
